src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -181,20 +181,20 @@
     parameters:
         - name: --source -s
           type: string
-          short-summary: The directory where the files to be uploaded.
+          short-summary: The directory from which the files should to be uploaded.
         - name: --destination -d
           type: string
-          short-summary: The string represents the destination of this upload operation. The source
-                         can be the container URL or the container name. When the source is the
-                         container URL, the storage account name will parsed from the URL.
+          short-summary: The string represents the destination of this upload operation. The
+                         destination can be the file share URL or the share name. When the source
+                         is the share URL, the storage account name will parsed from the URL.
```

Comment:
Does the "file share URL" also account for directory path? My URL could be "https://file.account.azure.com/myshare/dir1/dir2/dir3". That was supported with the stand-alone upload. In this case the "path" parameter in the upload call would be "dir1/dir2/dir3".